packages/coinstac-common/src/models/decentralized-computation.js

```diff
@@ -25,14 +25,59 @@ class DecentralizedComputation extends Base {
    * Get a serialized model suitable for saving in the 'computations' database.
    *
    * @returns {Object} Serialized model
+   * @property {Object[]} [inputs]
    * @property {string} name
    * @property {string} url
    * @property {string} version
    */
   getComputationDocument() {
     const { name, repository: { url }, version } = this.serialize();
+    const doc = { name, url, version };
 
-    return { name, url, version };
+    /**
```

Comment:
These changes ensure a computation definition’s `inputs` are saved to the `DecentralizedComputation`’s document in the _computations_ database. It’s pretty rough: we should offer some sort of plugin for computations to save relevant info to their docs.